catalyst/dl/scripts/swa.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import argparse
+from argparse import ArgumentParser
+import logging
+from pathlib import Path
+
+from catalyst.dl.utils.swa import generate_averaged_weights
+
+
+def build_args(parser: ArgumentParser):
+    """Builds the command line parameters."""
+    parser.add_argument("logdir", type=Path, help="Path to models logdir")
+    parser.add_argument(
+        "--models_mask", "-m", type=str, help="Pattern for models to average"
+    )
+    parser.add_argument(
+        "--save-avaraged-model",
```

Comment:
averaged?

Reply:
why do we need this flag? this script should always produce averaged checkpoints, isn't it?